### PR TITLE
added task to create users with 4 different profiles

### PR DESCRIPTION
**Pull request by skristem**
# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

File: cumulusci.yml [hunk 1]
```diff
@@ -140,6 +140,14 @@ tasks:
             path: unpackaged/config/wave
             package_xml: manifests/wave.xml
 
+    create_users:
```
Comment (Julian88Tex) [inline cumulusci.yml]: @skristem maybe could be a bit more explicit as to what kind of users it's creating

```suggestion
    create_community_users:
```

File: cumulusci.yml [hunk 2]
```diff
@@ -140,6 +140,14 @@ tasks:
...
+        description: Creates Users with various profiles for automation
+        class_path: cumulusci.tasks.robotframework.Robot
+        options:
+            suites: tasks/create_users.robot
```
Comment (Julian88Tex) [inline cumulusci.yml]:
```suggestion
            suites: tasks/create_community_users.robot
```

File: cumulusci.yml [hunk 3]
```diff
@@ -140,6 +140,14 @@ tasks:
             path: unpackaged/config/wave
             package_xml: manifests/wave.xml
 
+    create_users:
+        description: Creates Users with various profiles for automation
```
Comment (Julian88Tex) [inline cumulusci.yml]: @skristem 
```suggestion
        description: Creates Community Users with various profiles for robot test automation
```

**Review by Julian88Tex — APPROVED** [review]
@skristem a few small suggestions just to be a bit more explicit but either way looks good thanks so much!

**Review by rjanjanam — APPROVED** [review]

**Pull request merged**